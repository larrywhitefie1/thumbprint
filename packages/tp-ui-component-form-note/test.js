const sass = require('node-sass');

it('compiles correctly', () => {
    const css = sass
        .renderSync({ file: 'packages/tp-ui-component-form-note/_index.scss' })
        .css.toString();
    expect(css).toMatchSnapshot();
});
