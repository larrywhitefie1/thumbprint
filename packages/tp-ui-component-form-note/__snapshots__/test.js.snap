// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compiles correctly 1`] = `
".tp-form-note {
  margin-top: 4px;
  font-size: 12px; }

.tp-form-note--bad-news {
  color: #ff5a5f; }
"
`;
